src/style/list/list.less

```diff
@@ -250,7 +250,7 @@ body.theme-dark {
   &:hover {
     .right-column > .icon-more {
       opacity: 1;
-      transition-delay: calc(var(--animation-timing-faster) + 0.05);
+      transition-delay: calc(var(--animation-timing-faster) + 0.05s);
```

Comment:
We have these variables for animation-timing:
```
  --animation-timing-faster: 0.15s;
  --animation-timing-fast: 0.35s;
```

Can we create a new one with `0.20s`?